src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -61,7 +61,17 @@ Then, you can see the error in your dashboard:
 [{% asset js-index/error-message.png alt="Error in Unresolved Issues with title This is my fake error message" %}]({% asset js-index/error-message.png @path %})
 
 ## Capturing Errors
-In most situations, you can capture errors automatically with `captureException()`.
+Automatically and manually capture errors, exceptions, and rejections.
+
+### Automatically Capture Errors
+By including and configuring Sentry, the SDK will automatically attach global handlers to capture uncaught exceptions and unhandled rejections.
+
+[{% asset js-index/automatically-capture-errors.png alt="Stack trace of a captured error" %}]({% asset js-index/automatically-capture-errors.png @path %})
+
+Browsers take security measures when serving script files from different origins. To ensure errors are routed to Sentry, configure CORS headers, and add appropriate script attributes. 
```

Comment:
I'd do something like this:

> Note: Browsers may take security measures when serving script files from different origins that can block error reporting. If you find errors are not being reported properly, please see this troubleshooting section on Cross Origin Resource Sharing (CORS).

... with that last bit linking to the troubleshooting guide you've added below.